src/components/Alert/index.tsx

```diff
@@ -41,8 +56,31 @@ const Alert = ({
       className={cs(styles.base, styles[variant], className, classNames?.root)}
       {...props}
     >
-      {Icon && <Icon className={cs(styles.icon, classNames?.icon)} />}
-      <span className={cs(classNames?.message)}>{message}</span>
+      <div className={styles.container}>
+        <div className={styles.content}>
+          <Icon className={cs(styles.icon, classNames?.icon)} />
+          <p className={cs(styles.message, 'eyebrow', classNames?.message)}>
+            {message}
+          </p>
+        </div>
+        {button ? (
+          <Button
+            type="secondary-neutral"
+            size="small"
+            onClick={onClose}
+            className={styles.button}
+          >
+            {button}
+          </Button>
+        ) : null}
+      </div>
+      <CloseButton
+        type="button"
+        className={cs(styles.closeButton, classNames?.closeButton)}
+        aria-label="Close"
+      >
+        {closeButton}
```

Comment:
if closeButton isn't defined or is null, shouldn't we hide this `CloseButton` element?

Reply:
We do not have a variant without a close button in the designs, all these global alerts have the close button X

Reply:
it is optional in the props (`closeButton?: React.ReactNode`) so you should either remove the `?` or handle the optional case